swagger_server/itm/itm_action_handler.py

```diff
@@ -230,13 +234,21 @@ def apply_treatment(self, action: Action, character: Character):
         # If the treatment treats the injury at the specified location, then change its status to treated.
         supply_used = action.parameters.get('treatment', None)
         attempted_retreatment = False
-        for injury in character.injuries:
-            if injury.location == action.parameters.get('location', None):
-                if injury.status != InjuryStatusEnum.TREATED: # Can't attempt to treat a treated injury
-                    if self._proper_treatment(supply_used, injury.name, injury.location):
-                        injury.status = InjuryStatusEnum.TREATED
-                else:
-                    attempted_retreatment = True
+        doesnt_treat_injuries = [SupplyTypeEnum.BLANKET, SupplyTypeEnum.BLOOD, SupplyTypeEnum.EPI_PEN, \
```

Comment:
Curious about the inclusion of some supplies here, is it just because we haven't defined the specific injuries yet?  I would assume blood would treat an injury or condition of "loss of blood" and that an epi pen would treat an allergic reaction.  Is this just a temporary hold spot for those until the rest is implemented?